src/plugins/intel_cpu/src/config.cpp

```diff
@@ -320,48 +358,50 @@ void Config::updateProperties() {
     if (!_config.empty())
         return;
 
-    using namespace InferenceEngine;
     switch (streamExecutorConfig._threadBindingType) {
     case IStreamsExecutor::ThreadBindingType::NONE:
-        _config.insert({ PluginConfigParams::KEY_CPU_BIND_THREAD, PluginConfigParams::NO });
+        _config.insert({ov::internal::cpu_bind_thread.name(), "NO"});
```

Comment:
@riverlijunjie @wangleis Why we keep using values which are not part of `ThreadBindingType`? Do we have plans to refactor this?

Reply:
It is a gap between cpu plugin and stream executor, we need refactor IStreamsExecutor::Config, so I didn't put the change into this PR.

Reply:
TODO: CVS-126094